requirements.txt

```diff
@@ -4,5 +4,5 @@ requests>=2.24
 setuptools
 sphinx
 tomli>=1.2.3; python_version<'3.11'
-urllib3>=1.26.5,<2.0.0
+urllib3
```

Comment:
[Renovate wants this to be `urllib3>=2.0.3,<2.1.0`.](https://github.com/containers/podman-py/pull/287/files)

IMHO, it's probably a good idea to pin this to a range.  W/o pinning, an incompatible/breaking change can happen and there'll never be a podman-py CI failure to let us know.  It'll simply break without notice.

Reply:
JFYI, this could potentially exclude urllib3 version that are used on centos. Otherwise I'm completely fine with adding this range.